### DISCUSSION TITLE
How i can get previus and next slide in the slideChangeTransitionEnd event?

### DISCUSSION THREAD

**Discussion by xkomiks** [nolimits4web/swiper#4004] [Q&A]

**Comment by vltansky** [comment]
What exactly do you need? Index?

https://swiperjs.com/api/
`mySwiper.activeIndex`
`mySwiper.previousIndex`

**Comment by xkomiks** [comment]
I need a DOM slide element

**Comment by vltansky** [comment]
First of all, it's a question for stack overflow, not for the GitHub issue.
Second there are plenty ways to get what you want: simply select `.swiper-next` / `.swiper-prev`:
`document.querySelector('.swiper-slide-next')`
or 
```js
swiper.on('_slideClass',, (_, el, classNames) => {
  if(classNames.indexOf('swiper-slide-next') >= 0){
   console.log(el);
  }
});

**Comment by vltansky** [comment]
@kravetsIn issue can be closed?
